rhp/v2/rpc_test.go

```diff
@@ -355,7 +355,7 @@ func TestRenew(t *testing.T) {
 		}
 		// upload the sector
 		remainingDuration = contractExpiration - currentHeight
-		price, collateral := rhp2.RPCAppendCost(session.Settings(), remainingDuration)
+		price, collateral := rhp2.RPCAppendCost(*session.Settings(), remainingDuration)
```

Comment:
the pointers fix themselves here when upgrading the `core` package which uses pointer receivers